### PR TITLE
Upgrade QCXF to 3.15.3

### PR DESCRIPTION
**Pull request by JiriOndrusek**
Backported https://github.com/apache/camel-quarkus/pull/6781

### CODE REVIEW

**Comment by JiriOndrusek** [comment]
We need to wait for the PR u[stream to get merged.

File: pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
         <optaplanner.version>9.37.0.Final</optaplanner.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
         <quarkiverse-amazonservices.version>2.18.1</quarkiverse-amazonservices.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/io/quarkiverse/amazonservices/quarkus-amazon-services-parent/ -->
         <quarkiverse-artemis.version>3.4.2</quarkiverse-artemis.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-parent/ -->
-        <quarkiverse-cxf.version>3.15.2</quarkiverse-cxf.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/io/quarkiverse/cxf/quarkus-cxf-parent/ -->
+        <quarkiverse-cxf.version>3.15.3</quarkiverse-cxf.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/io/quarkiverse/cxf/quarkus-cxf-parent/ -->
         <quarkiverse-cxf-community.version>3.15.2</quarkiverse-cxf-community.version>
```
Comment (jamesnetherton) [inline pom.xml:58]:
```suggestion
        <quarkiverse-cxf-community.version>3.15.3</quarkiverse-cxf-community.version>
```

**Review by ppalaga — CHANGES_REQUESTED** [review]
I'd vote for backporting https://github.com/apache/camel-quarkus/pull/6782 rather than https://github.com/apache/camel-quarkus/pull/6781

**Comment by JiriOndrusek** [comment]
Sure, I'll do it

**Comment by JiriOndrusek** [comment]
Superseded by https://github.com/jboss-fuse/camel-quarkus/pull/459

**Pull request closed without merging**